packages/fxa-auth-server/scripts/verification-reminders.js

```diff
@@ -27,18 +27,35 @@ const verificationReminders = require(`${LIB_DIR}/verification-reminders`)(
   log,
   config
 );
+import * as Sentry from '@sentry/node';
```

Comment:
I hear mixing import and require is bad form.  I can remove this file from the patch if y'all don't like it.  This file was the only old JS one.